.github/workflows/trigger-chromium-build.yml

```diff
@@ -0,0 +1,131 @@
+
+                # Check for puppeteer_version property
+                if echo "$block_content" | jq -e 'has("puppeteer_version")' > /dev/null; then
+                  PUPPETEER_VERSION_COUNT=$((PUPPETEER_VERSION_COUNT + 1))
+                  if [[ $PUPPETEER_VERSION_COUNT -gt 1 ]]; then
+                    # Mark multiple puppeteer_version properties found, exit with error
+                    echo "multiple_puppeteer_version_properties=true" >> $GITHUB_OUTPUT
+                    exit 0
+                  fi
+                  UPGRADE_CONFIG="$block_content"
+                fi
+              fi
+          }
+
+          # Main function to extract code blocks from a markdown file
+          extract_code_blocks() {
+              mdsh-parse handle_code_block
+              # Output the upgrade config if found
+              if [[ -n "$UPGRADE_CONFIG" ]]; then
+                  echo "version_bump_config=$UPGRADE_CONFIG" >> $GITHUB_OUTPUT
+              fi
+          }
+
+          # attempt extracting puppeteer version bump config from the issue body, 
+          # single quote is very much intentional here so it's not escaped by bash
+          echo '${{ github.event.issue.body }}' | extract_code_blocks
+      - name: Report multiple puppeteer_version properties
+        if: ${{ steps.extract_version_bump_config.outputs.multiple_puppeteer_version_properties == 'true' }}
+        uses: actions/github-script@60a0d83039c74a4aee543508d2ffcb1c3799cdea #v7.0.1
+        with:
+          script: |
+            github.rest.issues.createComment({
+              issue_number: context.issue.number,
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              body: 'Multiple JSON blocks with puppeteer_version found. Please provide only one JSON block with the puppeteer_version property.'
+            })
+      - name: Report missing puppeteer_version properties
+        if: ${{ steps.extract_version_bump_config.outputs.result == '' }}
+        uses: actions/github-script@60a0d83039c74a4aee543508d2ffcb1c3799cdea #v7.0.1
+        with:
+          script: |
+            github.rest.issues.createComment({
+              issue_number: context.issue.number,
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              body: 'No puppeteer_version property found in the issue body. Please provide the puppeteer_version property in a JSON code block.'
+            })
+
+  trigger_build:
+    runs-on: ubuntu-latest
+    needs: [matches_label]
+    if: ${{ needs.matches_label.outputs.version_bump_config }}
+    steps:
+      - uses: elastic/oblt-actions/buildkite/run@v1
+        with:
+          token: ${{ secrets.SHAREDUX_BUILDKITE_TOKEN }}
+          pipeline: 'kibana-chromium-linux-build'
+          env-vars: |
+            GITHUB_PR_BASE_OWNER=${{ github.event.repository.owner.name }}
+            GITHUB_PR_BASE_REPO=${{ github.event.repository.name }}/
```

Comment:
The slash preceding the repo name is intentional

Reply:
are you passing these, because they're required by the `comment_on_pr.ts` script? 

that script was meant to be used in conjunction with PRs, that's why it expects these variables to be present, (also the `env:` section you declare in the top of the pipeline implementation). 

If you were to adjust the script to take the parameters optionally through arguments, then you will only need the issue number, everything else is set

Reply:
Yes... I passed it in because of the `comment_on_pr.ts` script, I didn't want to make too many changes especially if I could get it to work without doing so, I also understand that makes it's usage brittle. However since you're fine with this, I'll make some changes accordingly.

Reply:
Thanks! If you don't mind, I'd prefer to extend the usage in this PR rather than a separate that only adjusts the awkwardness - that would be forgotten anyway.

Reply:
Reworked this and created a `comment_on_issue.ts` script that leverages the internals `comment_on_pr.ts` in [2178951](https://github.com/elastic/kibana/pull/212674/commits/2178951ede89d90aa0afa0d363d0c9941be1a18f)